chart/values.yaml

```diff
@@ -52,7 +52,7 @@ base:
     containers:
       - name: etcd-probe
         image: busybox:latest
-        command: ['sh', '-c', 'trap "exit 1" TERM; until nc -vzw 5 {{ .Release.Name }}-etcd {{ .Values.etcd.service.port }}; do date; echo "Waiting for etcd..."; sleep 1; done;']
+        command: ['sh', '-c', 'trap "exit 1" TERM; until nc -vzw 5 {{ .Release.Name }}-etcd {{.Values.etcd.service.port}}; do date; echo "Waiting for etcd..."; sleep 1; done;']
```

Comment:
Is the space not needed?

Reply:
No Vishnu, the space causes lint issues.

Reply:
Fixing this helps us with picking the mayastor-extensions chart as-is without having to overwrite some things. The openebs/charts linter fails with these spaces. Typically the values yaml wasn't meant for templates, so doesn't have template support as much as the template directory stuff.